base/compiler/ssair/inlining.jl

```diff
@@ -428,14 +427,14 @@ function ir_inline_item!(compact::IncrementalCompact, idx::Int, argexprs::Vector
         just_fixup!(inline_compact)
         compact.result_idx = inline_compact.result_idx
         compact.active_result_bb = inline_compact.active_result_bb
-        for i = 1:length(pn.values)
-            isassigned(pn.values, i) || continue
-            v = pn.values[i]
-            if isa(v, SSAValue)
-                compact.used_ssas[v.id] += 1
-            end
-        end
         if length(pn.edges) == 1
+            for i = 1:length(pn.values)
```

Comment:
If this is necessary here, then so is counting the ssa value of the newly used instruction. However, since we're not adding any uses, I don't think it is and the count should happen when we actually insert the value during ssa rename.

Reply:
Yeah, I kinda assume that it will eventually be inserted (sort of why we're here, right?), but I wasn't completely sure yet. I think it's probably safe to just remove it altogether